### PR TITLE
Release v2.0.0beta

### PR DESCRIPTION
**Pull request by abulte**
WIP

Prep stuff for `v2.0.0` release:
- [x] changelog
- [x] add a migration helper function `migrate_from_1_to_2`
- [x] test admin interface after migrations
- [x] README w/ schema migration explanation
- [ ] test on dev env // integration w/ udata
- [ ] pypi (?)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -433,3 +433,4 @@ We’re using the [MIT license](https://tldrlegal.com/license/mit-license).
 ## Credits
 
 * [David Larlet](https://larlet.fr/david/)
+* [Alexandre Bulté](http://alexandre.bulte.net/)
```
Comment (davidbgk) [inline README.md]: 👍

**Comment by abulte** [comment]
@davidbgk this should be ready to ship from local to dev env — feel free to review!

File: croquemort/migrations.py [hunk 2]
```diff
@@ -145,3 +149,16 @@ def migrate_urls_redirect(self):
...
+    def migrate_from_1_to_2(self):
+        """
+        [migration from 1.0.0 to 2.0.0]
+
+        Helper to call all migrations needed from v1 to v2.
+        """
+        log('Migrating from 1 to 2...')
+        self.add_hash_prefixes()
+        self.migrate_urls_redirect()
+        log('Migration done.')
```
Comment (davidbgk) [inline croquemort/migrations.py]: MigrationS?

File: croquemort/webhook.py [hunk 3]
```diff
@@ -51,7 +51,8 @@ def _send(self, url, metadata):
     @event_handler('url_crawler', 'url_crawled')
     def send_response(self, metadata):
         """Call a webhook with checked url results"""
-        url = metadata.get('url')
+        # TODO test (and unit test)
```
Comment (davidbgk) [inline croquemort/webhook.py]: Still TODO?

Comment (abulte) [inline croquemort/webhook.py]: Good catch!

File: croquemort/tools.py [hunk 4]
```diff
@@ -111,8 +111,50 @@ def retrieve_datetime(datetime_isoformat):
 
 def is_url(url):
     # See https://github.com/kvesteri/validators for reference.
+    ip_middle_oct = u"(?:\.(?:1?\d{1,2}|2[0-4]\d|25[0-5]))"
+    ip_last_oct = u"(?:\.(?:[1-9]\d?|1\d\d|2[0-4]\d|25[0-4]))"
     url_pattern = re.compile(
-        r'^[a-z]+://([^/:]+\.[a-z]{2,10}|([0-9]{{1,3}}\.)'
-        r'{{3}}[0-9]{{1,3}})(:[0-9]+)?(\/.*)?$'
+        u"^"
```
Comment (davidbgk) [inline croquemort/tools.py]: That's quite a big change! Is there a [generic test suite](https://github.com/kvesteri/validators/blob/master/tests/test_url.py) that we can borrow somewhere? Or maybe use `validators` as a dependency?

In that case I prefer the dependency.

Comment (abulte) [inline croquemort/tools.py]: dependency 👍

**Comment by abulte** [comment]
@davidbgk all good for you?

**Pull request merged**